hyperspy/_components/arctan.py

```diff
@@ -53,6 +53,8 @@ class Arctan(Expression):
 
     def __init__(self, A=1., k=1., x0=1., module=["numpy", "scipy"], **kwargs):
         # Not to break scripts once we remove the legacy Arctan
+        if "legacy" in kwargs:
+            del kwargs["legacy"]
         if "minimum_at_zero" in kwargs:
             del kwargs["minimum_at_zero"]
```

Comment:
```suggestion
```

Reply:
as above, `'legacy'` removed, but `'mimimum_at_zero'` necessary for file reading